doc/index.rst

```diff
@@ -96,6 +96,7 @@ The Panel project is grateful for the sponsorship by the organizations and compa
    :hidden:
    :maxdepth: 2
 
+   Home <self>
```

Comment:
I was also hoping this was the issue but as far as I could tell this basically has no effect. We'll see in a dev build.